src/edu/ucsb/nceas/metacat/dataone/SystemMetadataFactory.java

```diff
@@ -100,6 +106,7 @@ public class SystemMetadataFactory {
 
 	public static final String RESOURCE_MAP_PREFIX = "resourceMap_";
 	private static Log logMetacat = LogFactory.getLog(SystemMetadataFactory.class);
+	private static final int wait_times = 4;
```

Comment:
Is this `index.resourcemap.waitingComponent.max.attempts` or should it be hardcoded?

Reply:
No, they are different. This is for morpho only. Since we will remove the support of Morpho soon, I decided not use new properties.